code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -194,6 +204,10 @@
 		var/organ = has_organ[index]
 		// organ new code calls `insert` on its own
 		new organ(H)
+	if(includes_special_organs)
+		for(var/I in special_organs)
+			var/obj/item/organ/internal/O = I
+			O.insert(H)
```

Comment:
I feel this would transfer over cyber organs and add them *in addition* to the organics that were just created a few lines above.

Reply:
This replaces the organs, IE if a cybernetic heart is here, it replaces the organic heart that would be generated